haskell/src/ruapu.h

```diff
@@ -0,0 +1 @@
+../../ruapu.h
```

Comment:
soft link may not work on windows

#include "../../ruapu.h" ?

Reply:
This is needed to satisfy the build system, referring a file out of the a package's root directory makes cabal fail to pack the it
Another choice may be simply remove the build script and document the usage in README, since this is really a small library and not difficult to build.

Reply:
As cabal have many issues with external include paths, I have removed the build script.